sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp

```diff
@@ -1,3 +1,11 @@
+//===----------------------------------------------------------------------===//
```

Comment:
nitpick,  but I see a lot of the existence license comments include the filename as part of this first comment line. Is that a convention we should be consistently following?

Reply:
Right. I think it's a common practice in LLVM project, but I don't know whether it's a requirement. Anyway, I'll add filenames.